### PR TITLE
[System Properties] LPS-154947 Replace method System.getProperty() with method SystemProperties.get() 

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I replace method System.getProperty() with method SystemProperties.get() in this pr.

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @zhangyan0701, Could you please help me scan test results when you free ? Thanks!

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:acceptance

**Comment by catarineaguiar** [comment]
no regressions on relevant, I'm waiting for acceptance results.

**Comment by catarineaguiar** [comment]
not possible to analyze acceptance results, because most tests were aborted. But according to relevant results no regression was found.

**Comment by vicnate5** [comment]
No regressions in the results we did get here. Passing QA review.

File: portal-impl/src/com/liferay/portal/util/PortalImpl.java [hunk 1]
```diff
@@ -318,14 +319,14 @@ public PortalImpl() {
 
 		// Computer name
 
-		String computerName = System.getProperty("env.COMPUTERNAME");
+		String computerName = SystemProperties.get("env.COMPUTERNAME");
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/util/PortalImpl.java:322]: @W2jx Your commit message is saying "replace system.properties's property", but here the env.* are not properties we set in system.properties.

**Comment by tinatian** [comment]
@W2jx The commit message `Replace method System.getProperty() with method SystemProperties.get() in classes that do not use the parameters of system.properties to avoid getting unresolved property values` is confusing, especially "the parameters of system.properties".

I believe here you want to say, there are some properties which do not have a default value defined in system.properties.

Tina.

**Comment by tinatian** [comment]
@W2jx Please see my comments and resend.

Thanks.
Tina

**Pull request closed without merging**